org.widelands.widelands.json

```diff
@@ -0,0 +1,478 @@
+                        "sed -r -i 's/^(\\[LIBXSLT_REQUIRED_VERSION=)/#\\1/' 'configure.ac';",
+                        "grep -Ei '_REQUIRED_VERSION' 'configure.ac' || :;"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'autoreconf';",
+                        "cp -p /usr/share/automake-*/config.{sub,guess} .;",
+                        "autoreconf -i;"
+                    ]
+                }
+            ],
+            "post-install": [
+                "echo 'post-install: symlink';",
+                "[[ -f '/app/opt/xmlstarlet/bin/xmlstarlet' ]] || ln -s 'xml' '/app/opt/xmlstarlet/bin/xmlstarlet';",
+                "echo 'post-install: docs';",
+                "install -d '/app/opt/xmlstarlet/share/doc/xmlstarlet'; install -p -m 0644 'AUTHORS' 'ChangeLog' 'INSTALL' 'NEWS' 'README' 'TODO' '/app/opt/xmlstarlet/share/doc/xmlstarlet/';",
+                "echo 'post-install: licenses';",
+                "install -d '/app/opt/xmlstarlet/share/licenses/xmlstarlet'; install -p -m 0644 'COPYING' 'Copyright' '/app/opt/xmlstarlet/share/licenses/xmlstarlet/';"
+            ],
+            "cleanup": [
+                "/opt/xmlstarlet/share/doc",
+                "/opt/xmlstarlet/share/man"
+            ]
+        },
+        {
+            "name": "imagemagick",
+            "no-parallel-make": true,
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static",
+                "--with-modules",
+                "--with-x",
+                "--with-threads",
+                "--with-magick_plus_plus",
+                "--with-gslib",
+                "--with-wmf",
+                "--with-webp",
+                "--with-openexr",
+                "--with-rsvg",
+                "--with-xml",
+                "--with-jbig",
+                "--with-openjp2",
+                "--prefix=/app/opt/imagemagick",
+                "--exec-prefix=/app/opt/imagemagick"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-6.9.9-44.tar.xz",
+                    "sha256": "4de6340f22563d80b93ee22b799c023d3a816d59be0b2fea8dd6ebc7b8fd4c7f"
+                },
+                {
+                    "type": "patch",
+                    "path": "ImageMagick-6.9.9-3-multiarch-implicit-pkgconfig-dir.patch",
+                    "strip-components": 1,
+                    "sha256": "80eea73e19a47cdb322d9006d4fe4a868db0d0d43fd7a0c9ae1a5e2fbbdbfe4b"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'copy examples';",
+                        "mkdir 'Magick++/examples'; cp -p Magick++/demo/*.cpp Magick++/demo/*.miff Magick++/examples;"
```

Comment:
Why do you install examples?

Reply:
This module comes from a different manifest, where it was desirable. In any case, it does not matter anyway, because at the end everything from `/opt` is removed. In addition, in this particular case I plan to replace whole `ImageMagick` package with the built-in `rsvg-convert` tool.